less/style.less

```diff
@@ -0,0 +1,408 @@
+/*fonts*/
+@import url('https://fonts.googleapis.com/css?family=Josefin+Sans:400,700');
+
+@font-face{
+	font-family: Aleo;
+	src: url(../fonts/Aleo-Font/Desktop-OTF/Aleo-Regular.otf);
+}
+
+/*colors*/
+
+@blue: rgb(84, 116, 253);
+@blue-button: rgb(73, 107, 254);
+@gray: rgb(51, 51, 51);
+@pink: #ff6363;
+
+/*common styles*/
+
+.logo-mix{
+	font-weight: 700;
+	font-family: 'Josefin Sans', sans-serif;
+	line-height: 26px;
+	color: #fff;
+	text-transform: uppercase;
+}
+
+.sub-logo{
+	font-weight: 400;
+	font-family: 'Josefin Sans', sans-serif;
+	line-height: 26px;
+	color: #fff;
+	text-transform: capitalize;
+}
+
+.border-box{
+	-webkit-box-sizing: border-box;
+	-moz-box-sizing: border-box;
+	box-sizing: border-box;
+}
+
+.h100{
+	height: 100%;
+}
+
+.flex{
```

Comment:
it isn`t block/separete class as it wouldn`t be BEM metodology 
it can be mixin 
the same for similar staff